packages/components/app/styles/components/tooltip.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// TOOLTIP & TOOLTIP-BUTTON
+//
+
+@use "../mixins/focus-ring" as *;
+
+:root {
+  // Dimensions:
+  --token-tooltip-max-width: 280px;
+
+  // Focus highlight position
+  --token-tooltip-tab-focus-offset: -2px;
+
+  // Colors:
+  --token-tooltip-surface-primary: var(--token-color-palette-neutral-700);
+}
+
+.hds-tooltip-button {
+  @include hds-focus-ring-with-pseudo-element(
+    $top: var(--token-tooltip-tab-focus-offset),
+    $right: var(--token-tooltip-tab-focus-offset),
+    $bottom: var(--token-tooltip-tab-focus-offset),
+    $left: var(--token-tooltip-tab-focus-offset),
+  );
+  margin: 0;
+  padding: 0;
+  color: inherit;
+  background: transparent;
+  border: none;
+
+  > .flight-icon { display: block; }
+}
+
+// this is the default html button display
```

Comment:
the fact that internally is a `<button>` is not relevant for the consumers, so we should not be too attached to it (the name of a component is not necessarily indicative of which HTML element is rendered: `Hds::SideNav::Header::IconButton` generates an `<a>` element if the `@href` argument is provided, for example; same for other components)

Reply:
It was just a comment on what the default display of a button element was. Anyway I'll delete it since I'm changing the display back to inline-flex